nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/Cluster.java

```diff
@@ -614,6 +615,7 @@ class Decorator implements CreationDecorator<Cluster> {
         public void decorate(AdminCommandContext context, final Cluster instance) throws TransactionFailure, PropertyVetoException {
             Logger logger = ConfigApiLoggerInfo.getLogger();
             LocalStringManagerImpl localStrings = new LocalStringManagerImpl(Cluster.class);
+            Random random = new SecureRandom();
```

Comment:
* Is it nessessary to use a SecureRandom for this? SecureRandom might block if the underlying system runs out of random byte input.
* wouldn't it be sufficient to have a static Random member

Reply:
You are right, just `Random` is really much faster (I tested via JMH).
This place is executed during cluster-related commands, e.g. once only. So for this usage, I suggest to leave the original SecureRandom.